src/feed/entities/feed.entity.ts

```
@@ -0,0 +1,19 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn } from "typeorm";
```

ESLint

[error] Replace `·Entity,·PrimaryGeneratedColumn,·Column,·CreateDateColumn·}·from·"typeorm"` with `⏎··Entity,⏎··PrimaryGeneratedColumn,⏎··Column,⏎··CreateDateColumn,⏎}·from·'typeorm'`

(prettier/prettier)